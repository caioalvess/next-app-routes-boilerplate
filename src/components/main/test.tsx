import { render, screen } from "@testing-library/react";

import Main from ".";

describe("<Main />", () => {
  it("should render the heading", () => {
    // render the component
    const { container } = render(<Main />);

    // search for the element and check its existence
    expect(
      screen.getByRole("heading", { name: /next app routes boilerplate/i }),
    ).toBeInTheDocument();

    // generate snapshot
    expect(container.firstChild).toMatchSnapshot();
  });
});
