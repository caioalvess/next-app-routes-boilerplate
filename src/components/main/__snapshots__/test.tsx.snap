// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Main /> should render the heading 1`] = `
<main
  class="sc-aYaIB fAXXLU"
>
  <h1>
    Next app routes boilerplate
  </h1>
</main>
`;
